packages/telegram-cloud-storage-stamper/README.md

```diff
@@ -0,0 +1,195 @@
+const telegramCloudStorageKey = "telegramCloudStorageKey";
+
+// get the item stored in Telegram Cloud Storage returned as a string
+const item = await stamper.getItem(telegramCloudStorageKey);
+
+if(!item) {
+  // failed retrieving item
+}
+
+```
+
+Insert a new API key into Cloud Storage at a specified key. This is just storing an API key, without using `.setSigningKey()` the key will not be used for signing. 
+
+```ts
+import { TelegramCloudStorageStamper } from "@turnkey/telegram-cloud-storage-stamper";
+
+// create a new Telegram Cloud Storage Stamper
+const stamper = new TelegramCloudStorageStamper();
+
+if (!stamper) {
+  // There was a problem initializing the stamper
+}
+
+const apiPublicKey = "...";
+const apiPrivateKey = "...";
+
+// the key used to index Telegram Cloud Storage
+const telegramCloudStorageKey = "telegramCloudStorageKey";
+
+// insert the API key in Telegram Cloud Storage
+const stored = await stamper.insertAPIKey(apiPublicKey, apiPrivateKey, telegramCloudStorageKey);
+
+if(!item) {
+  // failed storing API key
+}
+
+```
+
+Set a new API key as the signing key for the stamper at a specified key. This will also insert the API key to that location within Telegram CloudStorage. Any subsequent requests for stamping will sign with this API key. The API key and key can also be omitted and the API key at the default location `TURNKEY_API_KEY` will be used. If an API key is omitted and a key is specified an API key at that location will be used. Refer to [this](#argument-usage) section for a full explanation. A stamper that was originally used to just view Cloud Storage values can later be used for signing by using the `.setSigningKey()` function.
+
+```ts
+import { TelegramCloudStorageStamper } from "@turnkey/telegram-cloud-storage-stamper";
+
+// create a new Telegram Cloud Storage Stamper
+const stamper = new TelegramCloudStorageStamper();
+
+if (!stamper) {
+  // There was a problem initializing the stamper
+}
+
+const apiPublicKey = "...";
+const apiPrivateKey = "...";
+
+// the key used to index Telegram Cloud Storage
+const telegramCloudStorageKey = "telegramCloudStorageKey";
+
+// insert the API key in Telegram Cloud Storage
+const stored = await stamper.setSigningKey(apiPublicKey, apiPrivateKey, telegramCloudStorageKey);
+
+if(!item) {
+  // failed setting signing key
+}
+
+```
+
+Set a new API key as the signing key for the stamper that previously had a different key set for the stamper.
+
+```ts
+import { TelegramCloudStorageStamper } from "@turnkey/telegram-cloud-storage-stamper";
+
+// create a new Telegram Cloud Storage Stamper
+const stamper = await TelegramCloudStorageStamper.create({
+  apiPublicKey: "...",
+  apiPrivateKey: "...",
+});
+
+if (!stamper) {
+  // There was a problem initializing the stamper
+}
+
+const apiPublicKey2 = "...";
+const apiPrivateKey2 = "...";
+
+// insert the API key in Telegram Cloud Storage
+const stored = await stamper.setSigningKey(apiPublicKey2, apiPrivateKey2);
+
+if(!item) {
```

Comment:
`item` is undefined in this code snippet :)

Reply:
updated the readme in a few places for this